symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/CommandActivityInfo.java

```diff
@@ -0,0 +1,22 @@
+package com.symphony.bdk.core.activity.command;
+
+import com.symphony.bdk.core.activity.model.ActivityInfo;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apiguardian.api.API;
+
+/**
+ * {@link com.symphony.bdk.core.activity.command.CommandActivity} information/documentation model.
+ */
+@Getter
+@Setter
+@Accessors(fluent = true)
+@API(status = API.Status.EXPERIMENTAL)
+public class CommandActivityInfo extends ActivityInfo {
+
+  private String commandName;
```

Comment:
do we use it?

Reply:
I will reuse the `name` and the `description` of the `ActivityInfo`. So we can completely remove this class